cmd/generate.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Copyright 2020 The Swarm Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+# This script generates bmt hashes for all possible input lengths
+
+# prepare working directory
+wd=$(realpath $(dirname ${BASH_SOURCE[0]}))
+pushd $wd > /dev/null
+mkdir -p .data
+go build -o ./_bmt ./main_legacy.go
```

Comment:
i don't understand, why should compilation be a part of a data generation script?

Reply:
The script doesn't assume the binary installed to a PATH. Do you think it should?

Reply:
i'd assume that the script does a certain action (transform input/output) and does not have dependencies of, for example, build environment configuration (installing go at a certain version, for example).

with `cmd/generate` you're using naming conventions used by go project structure, but for a shell script. it would be nice if this command is just a go code which is compiled to a binary. i 

it would be nice if this is self contained in a go built binary. if you insist on having this as a shell script then i would say assume binary is in `PATH` and have a short `Makefile` that makes sure that this binary is built and installed correctly.
Personally I think this is not needed and that a self contained binary would suffice.

my 2c

Reply:
Yes, a complete implementation in go may be the most clean solution. To have `cmd/generate-hashes` or something more nicely named as the main package that would include code from main_legacy.go and logic from the shell script. This could be even invoked with `go run github.com/ethersphere/bmt/cmd/generate-hashes` to do generation, if someone would like to avoid making binary.